app/controllers/sites/forums_controller.rb

```diff
@@ -2,7 +2,7 @@ class Sites::ForumsController < Sites::BaseController
   activate_menu :audience, :forum, :settings
 
   before_action :authorize_forum_feature, :find_settings
-  before_action :active_settings_menu, unless: :forum_enabled?
```

Comment:
This `before_action` should be removed completely, because it activates the menu `:audience, :cms, :forum_settings` that is already removed here https://github.com/3scale/porta/pull/3602/files#diff-ac12a034e5e76892ef8e4d01b6b64340355bddc726088fc858bd45517dd30d54L170

Well, the whole controller doesn't make sense after forum removal, but it's OK to go step by step :grimacing: